### ISSUE TITLE
Lacks preview?

### ISSUE THREAD

**Issue by intelligence** [sanity-io/sanity-plugin-mux-input#189]
Using this in my schema I get a very non descriptive preview. As far as I can see there's no way to grab and use filename (or thumbnail?) did the API change?

<img width="597" alt="Skärmavbild 2023-01-20 kl  15 40 18" src="https://user-images.githubusercontent.com/756582/213725655-db6a38a3-a566-4d55-b135-1a38d1dbc772.png">

**Comment by wellermiranda** [comment]
I have the same problem:
<img width="654" alt="image" src="https://user-images.githubusercontent.com/961539/214414702-1d3fe308-bef2-430e-b4be-023257a3f181.png">

**Comment by stipsan** [comment]
Yes it seems like we have a regression. This was one of the first plugins we migrated to v3 after we started this work before last summer.
As the v3 APIs stabilised during the `npm i sanity@dev-preview` development cycle we must've missed updating this plugin with those changes.
It's supposed to show thumbnails that uses the thumbnail timecode, if defined, much like the "Browse" dialog: 
<img width="447" alt="image" src="https://user-images.githubusercontent.com/81981/214896582-d65278b8-5bd3-4c12-9aca-e3670f02fd75.png">

And for list previews it's indeed intended to show the original upload filename 🤔

**Comment by intelligence** [comment]
@stipsan Is this being worked on? Can I do anything to help out?

**Comment by mikehwagz** [comment]
I'd be happy to help out here as well if possible! Trying to get these previews fixed in a project asap

**Comment by TomasCarlson** [comment]
Have to work around this currently by making an object. Would be great if this is smoothed!

**Comment by patrulea** [comment]
How was this solved? I’m having trouble selecting the asset as the thumbnail of an object, just like @wellermiranda.

```js
preview: {
  select: {
    image: "image",
    video: "video",
  },
  prepare: ({ image, video }) => ({
    media: image || video
  })
}
```

![image](https://github.com/sanity-io/sanity-plugin-mux-input/assets/33817380/8f98b0c6-1727-4232-a75b-44b6099d3584)

(3rd entry is a video)
